packages/create-bison-app/template/jest.config.js

```diff
@@ -27,14 +27,14 @@ module.exports = {
   globals: {
     'ts-jest': {
       tsconfig: {
-        jsx: 'react',
+        jsx: 'react-jsx',
```

Comment:
This fixes an error that was thrown by removing all the explicit `React` imports:

> ‘React’ refers to a UMD global, but the current file is a module. Consider adding an import instead